tests/vexflow_test_helpers.ts

```diff
@@ -198,6 +198,7 @@ export class VexFlowTests {
   static FONT_STACKS: Record<string, string[]> = {
     Bravura: ['Bravura', 'Gonville', 'Custom'],
     Gonville: ['Gonville', 'Bravura', 'Custom'],
+    Gootville: ['Gootville', 'Bravura', 'Custom'],
```

Comment:
How does adding Gootville impact our test page's loading times? 

What about grunt test and `grunt generate:current`? Do they take much more time to run when we include another font?

I'm wondering if someday, we have 10 or 20 SMuFL fonts.... will our testing process become too cumbersome?

Reply:
`time npm run test:reference` 
**without Gootville**
```
real    1m7,523s
user    2m27,025s
sys     0m20,205s
```
**with Gootville**
```
real    1m16,106s
user    2m33,854s
sys     0m24,427s
```